### PR TITLE
add Go plugin

### PR DESCRIPTION
**Pull request by starius**
Plugin description lives in [README.md](https://github.com/LuaAndC/mxe/blob/go/plugins/go/README.md).

### CODE REVIEW

File: plugins/go/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+the [GOPATH](https://golang.org/doc/code.html#GOPATH) environment variable
+and call `usr/bin/$(TARGET)-go` wrapper script.
+
+## Example
+
+Building [gohs](https://github.com/flier/gohs), GoLang Binding of
+[HyperScan](https://01.org/hyperscan).
+
+```
+$ MXE_PLUGIN_DIRS=plugins/go make hyperscan go
```
Comment (tonytheodore) [inline plugins/go/README.md]: change to make variable instead of env: `make hyperscan go MXE_PLUGIN_DIRS=plugins/go`

**Comment by tonytheodore** [comment]
Looks good! Small fix for darwin https://github.com/LuaAndC/mxe/pull/7

File: plugins/go/README.md (hunk 1 above)
Comment (starius) [inline plugins/go/README.md]: Done.

**Comment by tonytheodore** [comment]
Thanks!

**Pull request merged**